tgui/packages/tgui/interfaces/Autolathe.js

```diff
@@ -0,0 +1,272 @@
+                </Tabs.Tab>
+              ))}
+            </Tabs>
+          </Stack.Item>
+        )}
+        <Stack.Item>
+          <Input
+            fluid
+            value={currentSearch}
+            placeholder="Search for an item"
+            onInput={(e, value) => setSearch(value.toLowerCase())}
+          />
+        </Stack.Item>
+        <Stack.Item grow>
+          <Flex direction="column">
+            {filteredPrintables.map((val, index) => {
+              return (
+                <Flex.Item key={index}>
+                  <Box>
+                    <Flex direction="row">
+                      <Flex.Item grow>
+                        <Button
+                          fluid
+                          content={capitalize(val.name)}
+                          disabled={!val.can_make}
+                          color={val.hidden ? 'red' : null}
+                          onClick={() =>
+                            act('make', {
+                              index: val.index,
+                              multiplier: 1,
+                            })
+                          }>
+                          {' (' + // sorry for this shitcode, also yes this will break if an autolathe uses more than 2 material types
+                            (val.materials[Object.keys(materials)[0]] &&
+                            val.materials[Object.keys(materials)[1]]
+                              ? val.materials[Object.keys(materials)[0]] +
+                              ', ' +
+                              val.materials[Object.keys(materials)[1]]
+                              : val.materials[Object.keys(materials)[0]]
+                                ? val.materials[Object.keys(materials)[0]]
+                                : val.materials[Object.keys(materials)[1]]) +
+                            ') '}
+                        </Button>
+                      </Flex.Item>
+
+                      {(val.has_multipliers && (
+                        <Fragment>
+                          <Box width="2.5px" />
+                          <Flex.Item>
+                            <Flex direction="row">
+                              {Object.keys(val.multipliers).map((entry) => (
+                                <Flex.Item key={entry}>
+                                  <Button
+                                    content={'x' + entry}
+                                    onClick={() =>
+                                      act('make', {
+                                        index: val.index,
+                                        multiplier: entry,
+                                      })
+                                    }
+                                  />
+                                </Flex.Item>
+                              ))}
+                            </Flex>
+                          </Flex.Item>
+                        </Fragment>
+                      )) ||
+                        null}
```

Comment:
This seems like it isn't required

Reply:
if I don't use the || null it puts this weird plaintext zero after the button for ones with no multipliers